cmd/main_postsubmit.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func runCommand(root string, runMake bool, path string, releaseBranch string, release string, artifactBucket string, args []string) {
```

Comment:
Then this guy, the arguments just need to be converted to get from `cmd` and it becomes `func (cmd *Command) run(runMake bool, path string) int {`